documentation/docs/pipelines/abapEnvironment/introduction.md

```diff
@@ -46,8 +46,15 @@ After the confirmation, the Communication Arrangement [SAP_COM_0510](https://hel
 
 ## Clone Repositories
 
+If the `Prepare System` stage is not executed, we assume that a system is already configured and all Software Components are cloned.
 In this stage, the Software Components / Git repositories are pulled to the ABAP Environment system using the step abapEnvironmentPullGitRepo.
 The step can receive a list of Software Components / repositories and pulls them successively.
+Please refer to the Configuration section for the abapEnvironment Pipeline or the respective documentations for the abapEnvironmentCheckoutBranch and abapEnvironmentPullGitRepo steps.
+If the `Prepare System` stage is executed, the abapEnvironmentCloneGitRepo step will be performed instead.
```

Comment:
Needs to be updated

Reply:
Actually line 49 was fine but I changed it anyway

I mean the assumption still exists that the system should already be configured😎